pygmt/figure.py

```diff
@@ -284,7 +284,12 @@ def show(self, dpi=300, width=500, method=None):
         This is useful when running unit tests and building the documentation
         in consoles without a Graphical User Interface.
 
-        Note that the external viewer does not block the current process.
+        Note that the external viewer does not block the current process, thus
+        it's necessary to suspend the execution of the current process for a
+        short while after launching the external view, so that the preview
+        image won't be deleted before the external viewer tries to open it. Set
+        the ``waiting`` parameter to a larger number if your computer is old
+        and slow.
```

Comment:
```suggestion
        the ``waiting`` parameter to a larger number if your computer is slow.
```
I feel like "slow" is enough to let the user know to change the waiting parameter.